src/app/components/client/SignInButton.tsx

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { signIn, useSession } from "next-auth/react";
+import { useL10n } from "../../hooks/l10n";
+import { Button } from "./Button";
+
+export const SignInButton = () => {
+  const l10n = useL10n();
+  const session = useSession();
+
+  if (typeof session.data?.user.email === "string") {
+    return null;
+  }
+
+  return (
+    <Button
+      variant="secondary"
+      onPress={() =>
+        void signIn("fxa", { callbackUrl: "/redesign/user/dashboard/" })
+      }
```

Comment:
Q: Does this need telemetry, or do we capture clicks in the target page views anyways?

Reply:
Yep, Kaitlyn [asked](https://github.com/mozilla/blurts-server/pull/3990#discussion_r1446341879) as much as well! Added in ad04d880d.